python/README.md

```diff
@@ -33,3 +33,42 @@ Alternatively, install nox and run the tests inside an isolated environment:
 ```shell
 nox
 ```
+
+
+## Previewing Docs
+
+* Install `quarto-cli` on your machine. Also consider installing an IDE extension.
+
+  See: https://quarto.org/docs/get-started/
+
+* Generate reference docs
+
+  ```shell
+  nox -s gen-docs
+  ```
+
+  You should re-run this after making any updates to the `pysrc` docstrings.
+  If _Preview Docs_ is running in another shell, the system should auto-refresh with your changes.
```

Comment:
Does this mean that I need to run `nox -s gen-docs docs` to first generate the reference docs and then start a preview, but after I don't need to rerun `nox -s gen-docs` to update? If the later, would it make sense for `nox -s docs` to run the equivalent of `gen-docs` internally, so we only need to run a single command?

Reply:
you do need to run `nox -s  gen-docs` anytime you want to update the generated docs. this is why they weren't combined into a single command.

run `nox -s docs` in one shell and let it keep running.  make changes to anything outside generated docs, and it will auto refresh.  make changes to python docstrings, run `nox -s gen-docs` and then quarto will auto refresh.

there is also a command to watch for docstrings changes and auto refresh the generation of docs, but I wanted to limit the options a bit.  Also the two executables need to run separately.  I was looking for a way in nox to run two commands simultaneously in parallel, but that doesn't seem to be an option.

Reply:
the `docs-gen` will only re-write output files that have changed, so that the preview doesn't try to re-render the whole set of reference docs on each update. speeds up iterating on docs.